### PR TITLE
Change source package for gridengine on Ubuntu 1804

### PR DESCRIPTION
**Pull request by ddeidda**
Eoan (19.10) repo is no longer available. With this commit the package will be downloaded from Focal Fossa (20.04).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tilne** [comment]
Was the skip-changelog-update label added to temporarily prevent checks from failing while this PR is in draft? This change seems worthy of a changelog update to me.

File: attributes/default.rb [hunk 1]
```diff
@@ -302,7 +302,7 @@
   if node['platform_version'] == '18.04'
     default['cfncluster']['base_packages'].delete('libatlas-dev')
     default['cfncluster']['base_packages'].push('libatlas-base-dev', 'libssl-dev', 'libglvnd-dev')
-    default['cfncluster']['sge']['version'] = '8.1.9+dfsg-9build1'
+    default['cfncluster']['sge']['version'] = '8.1.9+dfsg-9build2'
```
Comment (lukeseawalker) [inline attributes/default.rb:305]: can we avoid pinning the build number?

**Review by lukeseawalker — APPROVED** [review]

File: attributes/default.rb (hunk 1 above)
Comment (ddeidda) [reply attributes/default.rb:305]: As discussed, we'll evaluate this approach in a further patch.

**Comment by ddeidda** [comment]
> Was the skip-changelog-update label added to temporarily prevent checks from failing while this PR is in draft? This change seems worthy of a changelog update to me.

Why do you think we should mention it? The version of SGE is still 8.1.9; the build number has never been mentioned and it's specific for Ubuntu.

**Pull request merged**